sklearn/preprocessing/_polynomial.py

```diff
@@ -957,7 +1014,13 @@ def transform(self, X):
         """
         check_is_fitted(self)
 
-        X = self._validate_data(X, reset=False, accept_sparse=False, ensure_2d=True)
+        X = self._validate_data(
+            X,
+            reset=False,
+            accept_sparse=False,
+            ensure_2d=True,
+            force_all_finite=False,
+        )
```

Comment:
> How to handle the use_sparse cases in transform, because scipy BSpline.build_matrix() doesn't accept any nan (see CI).

I think when `self.handle_missing == "indicator"`, you need to extract the missingness pattern here and you can replace the nans by an arbitrary non missing value (e.g. 0) prior to building the BSpline expansion.

Reply:
we decided to not have `handle_missing == "indicator"`, right?